### ISSUE TITLE
Watcher only watches one model

### ISSUE THREAD

**Issue by MaximBalaganskiy** [grofit/treacherous#13]
When I setup two validation groups on two separate models only the changes on the second model trigger the validation event. This happens because the validation group is created with the watcher singleton. The test code
```
        let ruleSet1 = createRuleset().forProperty("name1").addRule("required").build();
        let ruleSet2 = createRuleset().forProperty("name2").addRule("required").build();
        let model1 = { name1: "name1" };
        let model2 = { name2: "name2" };
        let vg1 = createGroup(model1, ruleSet1);
        vg1.propertyStateChangedEvent.subscribe((...args) => { console.log("vg1", args); }, () => { return true; });
        let vg2 = createGroup(model2, ruleSet2);
        vg2.propertyStateChangedEvent.subscribe((...args) => { console.log("vg2", args) }, () => { return true; });

        model1.name1 = ""; // no log message here
        model2.name2 = ""; 
```

**Comment by grofit** [comment]
That is most interesting, there should be a new watcher per group, will verify this in a test shortly and get back to you.

**Comment by MaximBalaganskiy** [comment]
[https://github.com/grofit/treacherous/commit/3cb01949278ef91b9565fc8577cb8a8685b29c8c](This) commit introduced it

**Comment by grofit** [comment]
This should be fixed now in 0.9.2, however the knockout plugin may need updating. Can you confirm you are happy then close the issue?

**Comment by MaximBalaganskiy** [comment]
Works with aurelia, thanks for quick fix
